tools/python/src/cnn_face_detector.cpp

```diff
@@ -78,18 +117,42 @@ class cnn_face_detection_model_v1
     net_type net;
 };
 
-
 // ----------------------------------------------------------------------------------------
 
 void bind_cnn_face_detection()
 {
     using boost::python::arg;
     {
     class_<cnn_face_detection_model_v1>("cnn_face_detection_model_v1", "This object detects human faces in an image.  The constructor loads the face detection model from a file. You can download a pre-trained model from http://dlib.net/files/mmod_human_face_detector.dat.bz2.", init<std::string>())
-        .def("cnn_face_detector", &cnn_face_detection_model_v1::cnn_face_detector, (arg("img"), arg("upsample_num_times")=0),
+        .def(
+            "__call__", 
+            &cnn_face_detection_model_v1::detect, 
+            (arg("img"), arg("upsample_num_times")=0),
             "Find faces in an image using a deep learning model.\n\
           - Upsamples the image upsample_num_times before running the face \n\
             detector."
+            )
+        .def(
+            "__call__", 
+            &cnn_face_detection_model_v1::detect_mult, 
+            (arg("imgs"), arg("upsample_num_times")=0), 
+            "takes a list of images as input returning a 2d list of mmod rectangles"
             );
     }
+    {
+    typedef mmod_rect type;
+    class_<type>("mmod_rect", "Wrapper around a rectangle object and a detection confidence score.")
+        .def_readwrite("rect",   &type::rect)
+        .def_readwrite("confidence", &type::detection_confidence);
+    }
+    {
+    typedef std::vector<mmod_rect> type;
+    class_<type>("mmod_rectangles", "An array of mmod rectangle objects.")
```

Comment:
There is a general pattern in the dlib python api that goes like,  object, objects (for vector<object>) and objectss (for vector<vector<object>>).  This code should follow the same pattern so the API is consistent.  So these things should be called, mmod_rect, mmod_rects, and mmod_rectss.